tests/selenium/options_test.py

```diff
@@ -16,7 +18,11 @@ class OptionsPageTest(pbtest.PBSeleniumTest):
 
     def select_domain_list_tab(self):
         self.driver.find_element_by_css_selector('a[href="#tab-tracking-domains"]').click()
-        self.driver.find_element_by_id('show-tracking-domains-checkbox').click()
+        try:
+            self.driver.find_element_by_id('show-tracking-domains-checkbox').click()
+        except (NoSuchElementException, ElementNotInteractableException, ElementNotVisibleException):
```

Comment:
Why do we need to catch all three exception types? What are the different scenarios?

Reply:
ElementNotInteractableException is returned by Gecko, ElementNotVisibleException is returned by Chromedriver. Thanks for catching that, because the NoSuchElementException is indeed not needed!